package.json

```diff
@@ -1,51 +1,35 @@
 {
-  "name": "todolist",
+  "name": "leaderboard",
   "version": "1.0.0",
-  "description": "> A simple  list of tasks app using ES6 and webpack.",
-  "private": true,
+  "description": "> The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service. ## Live Demo",
+  "main": "webpack.config.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "build": "webpack",
-    "start": "webpack serve --open"
+    "test": "echo \"Error: no test specified\" && exit 1"
   },
```

Comment:
- [OPTIONAL] It would be great to have `start` and `bulid` in your `scripts:` in order to be able to run your app using `npm run start`:

![Screen Shot 2022-07-29 at 1 36 23 PM](https://user-images.githubusercontent.com/83717285/181823406-41532585-a823-4955-84f6-45bd9271cea3.png)

Reply:
Thank you. I would like to understand more about what you said in this optional. What do you mean? what should I do in detail, please

Reply:
@Jeanbulambo sure!

Reply:
@Jeanbulambo my suggestion is to add the `start` and `build` scripts to your `package.json`, this is how it works:

Every time you run `npm run SOMETHING` it will search on your `package.json` file for the `SOMETHING` script, in your case you had these scripts in the `package.json` on the `dev` branch, but they were not present in the `function-bran` branch. So if anyone clones your repo and runs `npm run start` they will get an error like:

![Screen Shot 2022-07-29 at 4 38 27 PM](https://user-images.githubusercontent.com/83717285/181846469-f9420dcb-1b14-49e8-9e61-c06242e4036f.png)